src/main/groovy/com/bmuschko/gradle/docker/utils/IOUtils.groovy

```diff
@@ -23,4 +23,14 @@ final class IOUtils {
             // ignore
         }
     }
+
+    static void closeQuietly(OutputStream output) {
```

Comment:
I would just change the original definition to take in a `Closeable instance` instead of creating another method.